packages/docs/docs/changelog.md

```diff
@@ -1,6 +1,10 @@
 # Changelog
 
-## 0.7.2
+## 0.7.3
+
+- Update readme on npm.
+
+## [0.7.2](https://github.com/fixie-ai/ai-jsx/pull/221/commits)
```

Comment:
Please link to the actual commit, which in this case is 203574abdbdce22c876a0c5a3a94dcc093b753cb.

Reply:
ok fixed